packages/dd-trace/src/profiling/config.js

```diff
@@ -35,13 +37,15 @@ class Config {
     const version = coalesce(options.version, DD_VERSION)
     const functionname = process.env.AWS_LAMBDA_FUNCTION_NAME
     // Must be longer than one minute so pad with five seconds
-    const flushInterval = coalesce(options.interval, 65 * 1000)
+    const flushInterval = coalesce(options.interval, DD_PROFILING_UPLOAD_PERIOD * 1000, 65 * 1000)
```

Comment:
Should make sure `DD_PROFILING_UPLOAD_PERIOD` is actually a number. Probably would actually be a string here.

Reply:
Ok, will do that, I will be cleaner
But, just to be clear, the current implementation would work, right ?
* if DD_PROFILING_UPLOAD_PERIOD is a string representing a number `DD_PROFILING_UPLOAD_PERIOD * 1000` will be a number and `colaesce` will select it
* if DD_PROFILING_UPLOAD_PERIOD is a not string representing a number `DD_PROFILING_UPLOAD_PERIOD * 1000` will be a NaN and `coalesce` will skip it
Shouldn't the same conversion be done for `DD_PROFILING_UPLOAD_TIMEOUT` ?

Reply:
Technically, yes. But generally best to be explicit about types in JS as much as possible. Coercion can do strange things. 😅